### PR TITLE
Panel styling

### PR DESCRIPTION
**Pull request by Artyssee**

### CODE REVIEW

File: resources/assets/sass/_styles.scss [hunk 1]
```diff
@@ -66,6 +66,16 @@ i{
 .dark{
   background-color: $dark-transparent;
   margin: 0;
+  width: 100%;
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: What is `.dark` for? panels?

File: resources/assets/sass/_styles.scss [hunk 2]
```diff
@@ -66,6 +66,16 @@ i{
...
+  height: 100%;
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: height doesn't have anything to do with a dark panel, does it?

File: resources/assets/sass/_styles.scss [hunk 3]
```diff
@@ -66,6 +66,16 @@ i{
...
+  position: fixed;
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: position does not have anything to do with a color

File: resources/assets/sass/_styles.scss [hunk 4]
```diff
@@ -66,6 +66,16 @@ i{
...
+}
+
+.white-container-overlay
+{
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: incorrect use of brackets

File: resources/assets/sass/_styles.scss [hunk 5]
```diff
@@ -66,6 +66,16 @@ i{
 .dark{
   background-color: $dark-transparent;
   margin: 0;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+}
+
+.white-container-overlay
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: terrible class name.

File: resources/assets/sass/_styles.scss [hunk 6]
```diff
@@ -95,7 +105,6 @@ i{
 .quiz-button{
   margin-top: 5%;
 }
-
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: why are you removing a line that needs to be there?

File: resources/assets/sass/_styles.scss [hunk 7]
```diff
@@ -66,6 +66,16 @@ i{
 .dark{
```
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: if this is about panels, why is it in styles.scss?

File: resources/assets/sass/_usercreation.scss [hunk 8]
```diff
@@ -1,68 +1,62 @@
-.img-panel {
-  display: block;
-  max-width: 100%;
-}
 .panel-top {
-  margin-top: 2%;
+  margin-top: $userpanel-margin-top;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: move this to _panels.scss

File: resources/assets/sass/_usercreation.scss [hunk 9]
```diff
@@ -1,68 +1,62 @@
...
 }
 .user-creation-img {
-  margin: 0 auto !important;
-  display: block;
+  margin: $userpanel-usercreationimg-margin;
+  margin-top: $user-image-margin-top !important;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: IMPORTANT

File: resources/assets/sass/_usercreation.scss [hunk 10]
```diff
@@ -1,68 +1,62 @@
...
 }
 .label-alignment {
-  display: inline-block;
+  display: $userpanel-label-alignment-display;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is display a terrible variable name?

File: resources/assets/sass/_usercreation.scss [hunk 11]
```diff
@@ -1,68 +1,62 @@
...
 }
-//Panel style
 .panel-label {
-  margin: 0 auto;
-  text-align: center;
-  font-size: 20px;
-  color: white;
+  margin: $userpanel-label-margin;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 12]
```diff
@@ -1,68 +1,62 @@
...
+  text-align: $userpanel-label-textalign;
+  font-size: $userpanel-label-fontsize;
+  color: $userpanel-label-color;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 13]
```diff
@@ -1,68 +1,62 @@
...
 }
-//Panel style
 .panel-label {
-  margin: 0 auto;
-  text-align: center;
-  font-size: 20px;
-  color: white;
+  margin: $userpanel-label-margin;
+  text-align: $userpanel-label-textalign;
+  font-size: $userpanel-label-fontsize;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 14]
```diff
@@ -1,68 +1,62 @@
...
+  display: $userpanel-label-alignment-display;
 }
-//Panel style
 .panel-label {
-  margin: 0 auto;
-  text-align: center;
-  font-size: 20px;
-  color: white;
+  margin: $userpanel-label-margin;
+  text-align: $userpanel-label-textalign;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 15]
```diff
@@ -1,68 +1,62 @@
...
+  font-size: $userpanel-label-fontsize;
+  color: $userpanel-label-color;
+  margin-top: $userpanel-margin-top;
 }
-.panel
-{
-  background-color: hsla(0, 0%, 80%, 0.67) !important;
-  border-radius: none;
+.panel {
+  background-color: $userpanel-background-color;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 16]
```diff
@@ -1,68 +1,62 @@
...
+  border-radius: $userpanel-border-radius;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 17]
```diff
@@ -1,68 +1,62 @@
...
+  margin-bottom: $userpanel-margin-bottom;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 18]
```diff
@@ -1,68 +1,62 @@
...
 }
-.panel-topmargin {
-  margin-top: 20px;
+.panel-margins {
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: terrible classname.

Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: this needs to be in _panels.scss

File: resources/assets/sass/_usercreation.scss [hunk 19]
```diff
@@ -1,68 +1,62 @@
...
+  margin-top: $userpanel-panel-topmargin;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 20]
```diff
@@ -1,68 +1,62 @@
...
-  padding-top: 20px;
-}
-.login-register-background {
-  background: linear-gradient(
-  rgba(0, 0, 0, 0.5),
-  rgba(0, 0, 0, 0.5)
-  ), url('../../img/background/01.jpg');
-  background-size: cover;
+  background-color: $userpanel-heading-background-color;
+  padding-top: $userpanel-heading-paddnig-top;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: typo in padding

File: resources/assets/sass/_usercreation.scss [hunk 21]
```diff
@@ -1,68 +1,62 @@
...
 }
-//Problem with overwriting border-radius so changed it to transperant color. If anyone has a solution go ahead
-.user-panel-page-properties
-{
-  background-color: rgba(255, 255, 255, 0.07);
+.panel-border {
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 22]
```diff
@@ -1,68 +1,62 @@
...
+  box-shadow: $userpanel-panel-border-box-shadow;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_usercreation.scss [hunk 23]
```diff
@@ -1,68 +1,62 @@
...
+  border-radius: $userpanel-panel-border-radius;
```
Comment (megaxlr) [inline resources/assets/sass/_usercreation.scss]: why is this a variable if this is only going to be used once?

File: resources/assets/sass/_styles.scss (hunk 7 above)
Comment (Artyssee) [inline resources/assets/sass/_styles.scss]: Its not just panels its also for the homepage and Dashboard page

File: resources/assets/sass/_variables.scss [hunk 24]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
 // Panels
 $panel-default-heading-bg: #fff;
 
+//userpanel
+//--panel-top
+$userpanel-margin-top: 5%;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 25]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+//--usr-creation-img
+$userpanel-usercreationimg-margin: 0 auto;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 26]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+//--user-image-margin
+$user-image-margin-top: 5%;
+//--label-alignment
+$userpanel-label-alignment-display: inline-block;
+//--panel-label
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 27]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-margin: 0 auto;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 28]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-textalign: center;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 29]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-fontsize: 20px;
+$userpanel-label-color: white;
+//--panel
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
+$userpanel-border-radius: none;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: border-radius cannot have none. you pushed illegal syntax css

File: resources/assets/sass/_variables.scss [hunk 30]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-usercreationimg-margin: 0 auto;
+//--user-image-margin
+$user-image-margin-top: 5%;
+//--label-alignment
+$userpanel-label-alignment-display: inline-block;
+//--panel-label
+$userpanel-label-margin: 0 auto;
+$userpanel-label-textalign: center;
+$userpanel-label-fontsize: 20px;
+$userpanel-label-color: white;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 31]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+//--usr-creation-img
+$userpanel-usercreationimg-margin: 0 auto;
+//--user-image-margin
+$user-image-margin-top: 5%;
+//--label-alignment
+$userpanel-label-alignment-display: inline-block;
+//--panel-label
+$userpanel-label-margin: 0 auto;
+$userpanel-label-textalign: center;
+$userpanel-label-fontsize: 20px;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 32]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-color: white;
+//--panel
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 33]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-border-radius: none;
+$userpanel-margin-bottom: 5%;
+//--panel-topmargin
+$userpanel-panel-topmargin: 2%;
+//--panel-heading
+$userpanel-heading-background-color: rgba(#fff, 0) !important;
+$userpanel-heading-paddnig-top: 20px;
+//--panel-border
+$userpanel-panel-border-box-shadow: 0px 1px 1px grey;
+$userpanel-panel-border-radius: none;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: border radius cannot be none. you pushed illegal css

Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 34]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
+$userpanel-border-radius: none;
+$userpanel-margin-bottom: 5%;
+//--panel-topmargin
+$userpanel-panel-topmargin: 2%;
+//--panel-heading
+$userpanel-heading-background-color: rgba(#fff, 0) !important;
+$userpanel-heading-paddnig-top: 20px;
+//--panel-border
+$userpanel-panel-border-box-shadow: 0px 1px 1px grey;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 35]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-color: white;
+//--panel
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
+$userpanel-border-radius: none;
+$userpanel-margin-bottom: 5%;
+//--panel-topmargin
+$userpanel-panel-topmargin: 2%;
+//--panel-heading
+$userpanel-heading-background-color: rgba(#fff, 0) !important;
+$userpanel-heading-paddnig-top: 20px;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 36]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-fontsize: 20px;
+$userpanel-label-color: white;
+//--panel
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
+$userpanel-border-radius: none;
+$userpanel-margin-bottom: 5%;
+//--panel-topmargin
+$userpanel-panel-topmargin: 2%;
+//--panel-heading
+$userpanel-heading-background-color: rgba(#fff, 0) !important;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 37]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-margin: 0 auto;
+$userpanel-label-textalign: center;
+$userpanel-label-fontsize: 20px;
+$userpanel-label-color: white;
+//--panel
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
+$userpanel-border-radius: none;
+$userpanel-margin-bottom: 5%;
+//--panel-topmargin
+$userpanel-panel-topmargin: 2%;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 38]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-label-alignment-display: inline-block;
+//--panel-label
+$userpanel-label-margin: 0 auto;
+$userpanel-label-textalign: center;
+$userpanel-label-fontsize: 20px;
+$userpanel-label-color: white;
+//--panel
+$userpanel-background-color: rgba(204, 204, 204, 0.67);
+$userpanel-border-radius: none;
+$userpanel-margin-bottom: 5%;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 39]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+//--panel-topmargin
+$userpanel-panel-topmargin: 2%;
+//--panel-heading
+$userpanel-heading-background-color: rgba(#fff, 0) !important;
+$userpanel-heading-paddnig-top: 20px;
+//--panel-border
+$userpanel-panel-border-box-shadow: 0px 1px 1px grey;
+$userpanel-panel-border-radius: none;
+//--btn-center-position
+$userpanel-btn-center-position-text-align: center;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 40]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-btn-center-position-margin-top: 5px;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 41]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-btn-center-position-margin-bottom: 5px;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 42]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+//--btn-fullwidth
+$usepanel-btn-fullwidth-width: 100%;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 43]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-btn-fullwidth-margin-bottom: 5px;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 44]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+//--@media screen and (max-width: 351px)
+$userpanel-respbreak-res-display: block;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: horrible variable name

File: resources/assets/sass/_variables.scss [hunk 45]
```diff
@@ -43,6 +43,43 @@ $input-color-placeholder: lighten($text-color, 30%);
...
+$userpanel-respbreak-res-padding-bottom: 10px;
```
Comment (megaxlr) [inline resources/assets/sass/_variables.scss]: image

File: resources/views/auth/login.blade.php [hunk 46]
```diff
@@ -1,19 +1,29 @@
...
 @section('content')
+  <div class="dark">
 <div class="container">
     <div class="row">
         <div class="col-md-8 col-md-offset-2">
-            <div class="panel panel-default paneltop">
-                <div class="panel-heading"><strong>Login</strong></div>
-                <div class="panel-body">
+            <div class="panel-label">
+                <div class="panel-heading panelhead-transperant-styling"><strong>Login</strong></div>
```
Comment (megaxlr) [inline resources/views/auth/login.blade.php]: Language files?

File: resources/views/auth/login.blade.php [hunk 47]
```diff
@@ -24,10 +34,10 @@
                         </div>
 
                         <div class="form-group{{ $errors->has('password') ? ' has-error' : '' }}">
-                            <label for="password" class="col-md-4 control-label">Password</label>
+                            <!-- <label for="password" class="col-md-4 control-label">Password</label> -->
```
Comment (megaxlr) [inline resources/views/auth/login.blade.php]: Commented HTML? why not remove it

File: resources/views/auth/login.blade.php [hunk 48]
```diff
@@ -38,27 +48,19 @@
...
-
-                        <div class="form-group">
-                            <div class="col-md-8 col-md-offset-4">
-                                <button type="submit" class="btn btn-primary">
+                            <div class="col-md-10 col-md-offset-1">
+                                <button type="submit" class="btn btn-primary btn-fullwidth">
                                     Login
                                 </button>
-                                <a class="btn btn-primary" href="/register">
+                                <a class="btn btn-primary btn-fullwidth" href="/register">
```
Comment (megaxlr) [inline resources/views/auth/login.blade.php]: btn-fullwidth exists in bootstrap. btn-block

File: resources/views/auth/login.blade.php [hunk 49]
```diff
@@ -38,27 +48,19 @@
...
                                     Nog geen lid? Meld je aan!
```
Comment (megaxlr) [inline resources/views/auth/login.blade.php]: language files?

File: resources/views/auth/login.blade.php [hunk 50]
```diff
@@ -38,27 +48,19 @@
...
                                 </a>
-
                                 <a class="btn btn-link" href="{{ route('password.request') }}">
                                     Forgot Your Password?
```
Comment (megaxlr) [inline resources/views/auth/login.blade.php]: language files?

File: resources/views/auth/login.blade.php [hunk 51]
```diff
@@ -38,27 +48,19 @@
...
                                 </a>
+                                    <label>
+                                        <input type="checkbox" name="remember" {{ old('remember') ? 'checked' : '' }}> Remember Me
```
Comment (megaxlr) [inline resources/views/auth/login.blade.php]: Language files?

File: resources/views/auth/register.blade.php [hunk 52]
```diff
@@ -67,10 +71,9 @@
 
                         <div class="form-group">
                           <div class="loginbutton col-md-10 col-md-offset-1 btn-center-position">
-                            <span class="respbreak"><button type="submit" class="btn btn-primary btn-samewidth">Register</button></span>
-                            <button onclick="location.href='login';" class="btn btn-primary registerest btn-samewidth">Hier al geweest? Log dan in!</button>
+                            <span class="respbreak"><button type="submit" class="btn btn-primary btn-fullwidth">Register</button></span>
```
Comment (megaxlr) [inline resources/views/auth/register.blade.php]: such class exists in Bootstrap. btn-block

File: resources/views/auth/register.blade.php [hunk 53]
```diff
@@ -67,10 +71,9 @@
...
+                            <button onclick="location.href='login';" class="btn btn-primary registerest btn-fullwidth">Hier al geweest? Log dan in!</button>
```
Comment (megaxlr) [inline resources/views/auth/register.blade.php]: Language files?

Comment (megaxlr) [inline resources/views/auth/register.blade.php]: such class already exists in Bootstrap, btn-block

File: resources/views/home.blade.php [hunk 54]
```diff
@@ -9,61 +8,69 @@
...
-              <div class="quiz-title col-md-8 col-sm-8 col-xs-12">
-                <h5>{{$quiz->title}}</h5>
-                <small>{{$quiz->description}}</small>
-              </div>
-              <div class="col-md-2 col-sm-2 col-xs-12">
-                <a class="quiz-button btn btn-success pull-right" type="button" name="button" style="margin-right: 40%" href="{{url('/quiz/' . $quiz->id)}}">@lang('dashboard.admin.view')</a>
-              </div>
-              <div class="col-md-2 col-sm-2 col-xs-12">
-                <a class="quiz-button btn btn-info pull-right" type="button" name="button" style="margin-right: 40%" href="{{url('/quiz/' . $quiz->id . '/statistics')}}">@lang('dashboard.admin.statistics')</a>
+      @if (Auth::user()->admin===1)
```
Comment (megaxlr) [inline resources/views/home.blade.php]: ===1 is redundant.

File: resources/views/welcome.blade.php [hunk 55]
```diff
@@ -1,35 +1,32 @@
...
+    plugins: 'fontawesome noneditable',
+    toolbar: 'fontawesome',
+    extended_valid_elements: 'span[*]' });</script>
 
 <header>
   <center>
-  <div class="dark">
-    <img src="{{ asset('img\logo\logo-large.png') }}" width="500" class="img-responsive">
+    <img src="{{ asset('img/logo/logo-large.png') }}" width="500" class="img-responsive">
     <h1 class="title">Omdat het ook anders kan!</h1>
```
Comment (megaxlr) [inline resources/views/welcome.blade.php]: Language files?

File: resources/assets/sass/_styles.scss (hunk 2 above)
Comment (Artyssee) [inline resources/assets/sass/_styles.scss]: Yes it does

File: resources/assets/sass/_usercreation.scss (hunk 8 above)
Comment (Artyssee) [inline resources/assets/sass/_usercreation.scss]: Been there done that

File: resources/assets/sass/_styles.scss (hunk 2 above)
Comment (megaxlr) [inline resources/assets/sass/_styles.scss]: no it doesn't. get your CSS skills sorted out or get out.

**Pull request closed without merging**